src/doc/feature/savemgr.dox

```diff
@@ -0,0 +1,80 @@
+// vim:ft=doxygen
```

Comment:
de savemgr doxygen is nog niet toegevoegd aan de features page

Reply:
Dat weet ik, ik had deze fix bovenop [`loek/doxygen`](https://github.com/lonkaars/crepe/tree/loek/doxygen) gemaakt omdat ik daar al andere aanpassingen aan de SaveManager API had gemaakt toen ik zag dat de API inconsistent was. Als die branch gemerget wordt wordt deze pagina ook toegevoegd aan de features pagina, maar dat is buiten de scope van deze PR. Ik had ze alvast meegenomen zodat het gebruik van de SaveManager duidelijk is, en omdat ik niels al eerder een screenshot moest sturen van deze pagina.